kore/test/Test/Kore/Step/Simplification/Exists.hs

```diff
@@ -37,12 +37,20 @@ import           Test.Tasty.HUnit.Extensions
 
 test_simplify :: [TestTree]
 test_simplify =
-    [ [plain10, plain11] `simplifies` [plain10', plain11']            $ "\\or distribution"
-    , [top]              `simplifies` [top]                           $ "\\top"
-    , []                 `simplifies` []                              $ "\\bottom"
-    , [equals]           `simplifies` [quantifyPredicate equals]      $ "\\equals"
-    , [substForX]        `simplifies` [top]                           $ "discharge substitution"
-    , [substOfX]         `simplifies` [quantifySubstitution substOfX] $ "substitution"
+    [ [plain10, plain11] `simplifiesTo` [plain10', plain11']
```

Comment:
This is completely stylistic, but it might make sense to use `(&)` here instead of `($)`; then the cases would read like
```.hs
"\\or distribution"
    & [plain10, plain11] `simplifiesTo` [plain10', plain11']
```